### PR TITLE
Feature/hashlink target

### PR DESCRIPTION
**Pull request by m0rkeulv**
Adding HashLink as a compile target

### CODE REVIEW

File: common/src/com/intellij/plugins/haxe/config/HaxeTarget.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
   // Target     clFlag    Extension,  OutputDir,  OutputType              Description
 
   NEKO(         "neko",   ".n",       "neko",     OUTPUT_TYPE.FILE,       HaxeCommonBundle.message("haxe.target.neko")),
+  HASHLINK(     "hl",     ".hl",      "hl",       OUTPUT_TYPE.FILE,       HaxeCommonBundle.message("haxe.target.hl")),
```
Comment (EricBishton) [inline common/src/com/intellij/plugins/haxe/config/HaxeTarget.java:36]: Why not just use the HL already defined at line 45(46)?  And, if it's not used anywhere else, we should probably just have one.

File: src/META-INF/plugin.xml [hunk 2]
```diff
@@ -85,6 +85,7 @@
   <![CDATA[
       <p>Unreleased changes</p>
       <ul>
+        <li>Add HashLink target.hx</li>
```
Comment (EricBishton) [inline src/META-INF/plugin.xml:88]: .hx??  :)

**Comment by EricBishton** [comment]
Also, HXMLData.readProperty() has to be updated to include the new enum.

**Review by EricBishton — CHANGES_REQUESTED** [review]
Also, I think that HASHLINK will have to be added to OpenFLTarget.  (But not NMETarget. :smile:)

Maybe HaxeRunner.doExecute() as well.  Otherwise we try to run Neko.

**Comment by m0rkeulv** [comment]
haha looks like i was a bit to tired when i wrote this stuff :D

File: src/META-INF/plugin.xml (hunk 2 above)
Comment (m0rkeulv) [reply src/META-INF/plugin.xml]: fixed

File: common/src/com/intellij/plugins/haxe/config/HaxeTarget.java (hunk 1 above)
Comment (m0rkeulv) [reply common/src/com/intellij/plugins/haxe/config/HaxeTarget.java]: a bit  tired i guess, fixed now

**Comment by m0rkeulv** [comment]
> 
> 
> Also, I think that HASHLINK will have to be added to OpenFLTarget. (But not NMETarget. 😄)
> 
> Maybe HaxeRunner.doExecute() as well. Otherwise we try to run Neko.

openFL target started Hashlink when i tested, and NME and the other targets are left unchanged so i don't think it will try to run Neko, unless i am missing something.

**Review by EricBishton — APPROVED** [review]

**Pull request merged**